mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -1301,7 +1301,7 @@ LogicalResult BufferLoadOp::verify() {
     return emitOpError("Expected " + Twine(nDims) + " coordinates for load");
   auto memSpaceValueAttr =
       sourceType.getMemorySpace().dyn_cast_or_null<gpu::AddressSpaceAttr>();
-  if (memSpaceValueAttr == nullptr &&
+  if (memSpaceValueAttr != nullptr &&
```

Comment:
Okay, immediately bumped into this commit that resolved my questions. But I assume we also need to nullptr check beforehand? That we disallow none address space?